app/models/early_career_payments/slug_sequence.rb

```diff
@@ -79,6 +80,7 @@ def slugs
         remove_student_loan_country_slugs(sequence)
         trainee_teacher_slugs(sequence) if claim.eligibility.trainee_teacher_in_2021?
         sequence.delete("future-eligibility") unless claim.eligibility.trainee_teacher_in_2021?
+        sequence.delete("eligible-degree-subject") unless claim.for_policy(LevellingUpPremiumPayments)&.eligibility&.eligible_none_of_the_above?
```

Comment:
Nitpicking - do we need the `&.` on those calls?

Reply:
No, it was mostly a hack to get some tests passing. I'll update the specs.